praw/models/reddit/removal_reasons.py

```diff
@@ -26,17 +32,23 @@ class RemovalReason(RedditBase):
 
     STR_FIELD = "id"
 
-    def __eq__(self, other):
+    def __eq__(self, other: _RemovalReason) -> bool:
```

Comment:
How do standard library type hints handle this case? There might be a better way.

Reply:
I’m pretty sure the standard library is filled with _T from trying to look at builtins and other modules, as many of them don’t call themselves